themes/digital.gov/src/scss/new/guides/_guide-nav.scss

```diff
@@ -124,12 +74,24 @@
       }
     }
 
+    // * After first top level link, indent following subnav links to show visual hierarchy
+    & > li.usa-sidenav__item:not(:first-child) a:not(.usa-button) {
+      padding-left: 2rem;
+    }
+
+    // * Indent sublist links for current page to show visual hierarchy
+    ul.usa-sidenav__sublist li.usa-sidenav__item a:not(.usa-button) {
+      padding-left: 2.75rem;
+    }
```

Comment:
Added indentation styles to display sidenav link hierarchy for sublist items and subnav link "pages".